internal/controller/apiextensions/composite/fuzz_test.go

```diff
@@ -25,6 +25,11 @@ import (
 
 	"github.com/crossplane/crossplane-runtime/pkg/resource/fake"
 
+	"github.com/ghodss/yaml"
```

Comment:
Nit: We typically try to use https://pkg.go.dev/sigs.k8s.io/yaml instead. Pretty sure it's the same API.